ui/app/src/services/dependencies.ts

```diff
@@ -17,12 +17,49 @@
  *
  */
 
-import { get } from '../utils/ajax';
+import { catchApi1Error, get, postJSON } from '../utils/ajax';
+import { map } from 'rxjs/operators';
+import { Observable } from 'rxjs';
 
 export function fetchDependencies(siteId: string, items: any) {
   return get(`/studio/api/2/dependency/dependencies?siteId=${siteId}&paths=${items}`)
 }
 
+export function getSimpleDependencies(siteId: string, path: string): Observable<any> {
+  return postJSON(
+    `/studio/api/1/services/api/1/dependency/get-simple-dependencies.json?site=${siteId}&path=${path}`,
+    null
+  ).pipe(
+    map((response: any) => {
```

Comment:
- Type response appropriately (AjaxResponse)
- When would this not have the response.response?
  - Can't you simply pluck as usual?